generators/php.js

```diff
@@ -34,7 +35,7 @@ Blockly.PHP = new Blockly.Generator('PHP');
  * accidentally clobbering a built-in object or function.
  * @private
```

Comment:
this is just calling a function, right? what exactly is being marked private? we shouldn't have anything listed private on the exported object

Reply:
Good point--the private tag doesn't make sense here. It's the same in the other language generators, so I'm going to remove it for all five generators in a commit in my next pr.